src/main/java/com/giantshoulder/driver/Driver.java

```diff
@@ -2,6 +2,9 @@
 
 import org.openqa.selenium.chrome.ChromeDriver;
 
+/**
```

Comment:
He is not the author.

Reply:
I did not write the Driver, Logger and Runner classes.

Reply:
Fixed in 90a07093045641fbfedf1a5ba7d45d026dca47f9!